Cannot deserialize value of type `me.itzg.helpers.modrinth.model.ProjectType` from String "resourcepack"

### Describe the problem

The modrinth downloader seems to have a hard time with one of these projects. I'm not particularly sure which one.

### Container definition

```
services:
  minecraft:
      image: itzg/minecraft-server:java21
      tty: true
      stdin_open: true
      restart: "on-failure"
      ports:
        - "25565:25565"
        - "25575:25575"
      environment:
        TYPE: "FABRIC"
        EULA: true
        VERSION: LATEST
        MODRINTH_ALLOWED_VERSION_TYPE: alpha
        MODRINTH_DOWNLOAD_DEPENDENCIES: optional
        MODRINTH_PROJECTS: |
            datapack:animated-doors
            datapack:dynamic-torches
            datapack:health-indicator
            datapack:qraftys-japanese-villages
            datapack:refined-advancements
            datapack:veinminer-enchantment
            alternate-current
            conduits-prevent-drowned
            crashexploitfixer
            creeper-healing
            death-backup
            fabric-api
            fallingtree
            fastback
            ferrite-core
            floodgate
            followers-teleport-too
            fsit
            geyser
            healing-campfire
            invview
            krypton
            leashable-players
            mc-258859
            moonrise-opt
            nemos-blooming-blossom
            no-chat-reports
            no-hostiles-around-campfire
            player-pronouns
            projectile-ding
            rabbit-pathfinding-fix
            recipecooldown
            scaffolding-drops-nearby
            sessility
            simple-voice-chat
            snowball-and-egg-knockback
            snowballs-freeze-mobs
            spawnproof-villagers
            stackdeobf
            sticky-enchanting-lapis
            styled-chat
            styled-nicknames
            towns-and-towers
            villager-names-serilum
            voice-chat-interaction
            what-are-they-up-to
      volumes:
        - ./server:/data
```


### Container logs

```
[mc-image-helper] 03:12:41.686 ERROR : 'modrinth' command failed. Version is 1.40.2
me.itzg.helpers.errors.GenericException: Failed to parse response body into class me.itzg.helpers.modrinth.model.Project: Cannot deserialize value of type `me.itzg.helpers.modrinth.model.ProjectType` from String "resourcepack": not one of the values accepted for Enum class: [mod, modpack]
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 358] (through reference chain: me.itzg.helpers.modrinth.model.Project["project_type"])
	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:89)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
	at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:103)
		at reactor.core.publisher.Mono.block(Mono.java:1712)
		at me.itzg.helpers.modrinth.ModrinthCommand.getVersionsForProject(ModrinthCommand.java:258)
		at me.itzg.helpers.modrinth.ModrinthCommand.lambda$expandDependencies$4(ModrinthCommand.java:165)
		at java.base/java.util.stream.ReferencePipeline$7$1.accept(Unknown Source)
		at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
		at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
		at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
		at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.forEachRemaining(Unknown Source)
		at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
		at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
		at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
		at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
		at java.base/java.util.stream.ReferencePipeline$7$1.accept(Unknown Source)
		at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
		at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
		at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
		at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
		at me.itzg.helpers.modrinth.ModrinthCommand.processProjects(ModrinthCommand.java:128)
		at me.itzg.helpers.modrinth.ModrinthCommand.call(ModrinthCommand.java:99)
		at me.itzg.helpers.modrinth.ModrinthCommand.call(ModrinthCommand.java:42)
		at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
		at picocli.CommandLine.access$1500(CommandLine.java:148)
		at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2457)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2419)
		at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
		at picocli.CommandLine$RunLast.execute(CommandLine.java:2421)
		at picocli.CommandLine.execute(CommandLine.java:2174)
		at me.itzg.helpers.McImageHelper.main(McImageHelper.java:159)
Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `me.itzg.helpers.modrinth.model.ProjectType` from String "resourcepack": not one of the values accepted for Enum class: [mod, modpack]
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 358] (through reference chain: me.itzg.helpers.modrinth.model.Project["project_type"])
	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:67)
	at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:1959)
	at com.fasterxml.jackson.databind.DeserializationContext.handleWeirdStringValue(DeserializationContext.java:1245)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._deserializeAltString(EnumDeserializer.java:447)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._fromString(EnumDeserializer.java:304)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer.deserialize(EnumDeserializer.java:273)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:310)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
	at com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:2125)
	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1501)
	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:85)
	... 9 common frames omitted
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -50,7 +50,7 @@ RUN easy-add --var os=${TARGETOS} --var arch=${TARGETARCH}${TARGETVARIANT} \
   --var version=${MC_SERVER_RUNNER_VERSION} --var app=mc-server-runner --file {{.app}} \
   --from ${GITHUB_BASEURL}/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_{{.os}}_{{.arch}}.tar.gz
 
-ARG MC_HELPER_VERSION=1.40.2
+ARG MC_HELPER_VERSION=1.40.4
 ARG MC_HELPER_BASE_URL=${GITHUB_BASEURL}/itzg/mc-image-helper/releases/download/${MC_HELPER_VERSION}
 # used for cache busting local copy of mc-image-helper
 ARG MC_HELPER_REV=1

--- a/tests/setuponlytests/vanillatweaks_file/docker-compose.yml
+++ b/tests/setuponlytests/vanillatweaks_file/docker-compose.yml
@@ -3,7 +3,7 @@ services:
     restart: "no"
     image: ${IMAGE_TO_TEST:-itzg/minecraft-server}
     environment:
-      VANILLATWEAKS_FILE: /config/vt-datapacks.json,/config/vt-craftingtweaks.json,/config/vt-resourcepacks.json
+      VANILLATWEAKS_FILE: /config/vt-datapacks.json,/config/vt-craftingtweaks.json
       EULA: "TRUE"
       SETUP_ONLY: "TRUE"
       # the following are only used to speed up test execution
@@ -14,6 +14,5 @@ services:
       - ./data:/data
       - ./vt-datapacks.json:/config/vt-datapacks.json:ro
       - ./vt-craftingtweaks.json:/config/vt-craftingtweaks.json:ro
-      - ./vt-resourcepacks.json:/config/vt-resourcepacks.json:ro
       # the following are only used to speed up test execution
       - ./verify.sh:/servers/fake.jar

--- a/tests/setuponlytests/vanillatweaks_file/require.sh
+++ b/tests/setuponlytests/vanillatweaks_file/require.sh
@@ -1,0 +1,1 @@
+[[ $EXTENDED_TESTS ]] || exit 1

